### DISCUSSION TITLE
Mac agent not starting

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#5887] [Issues] [answered]
Hi all,

I've installed the Mesh on my Mac and it went good.
The agent is not restarting after restarting the machine.
I've tried runing _launchctl start meshagent_ in _/Library/LaunchAgents/_ from the terminal but it's not working.
How can I see in the service is even working via the terminal?
Is there also a way to start the agent automatically with a restart?

Thanks.

**Comment by si458** [comment, marked as answer]
this has been fixed recently, just need new agents being made and released
the work around is here https://github.com/Ylianst/MeshCentral/issues/5653#issuecomment-1963039734

**Reply by ghost** [reply]
Works, thanks.

**Reply by si458** [reply]
Something changed when ventura (cant temember which version) was released that each agent plist file needs it's own name now (go figure)
